typhon/objects/files.py

```diff
@@ -493,7 +493,7 @@ def writeSetContentsCB(fs):
         with ruv.unstashingFS(fs) as (vat, sc):
             assert isinstance(sc, SetContents)
             size = intmask(fs.c_result)
-            if size > 0:
+            if size >= 0:
```

Comment:
Actually, while you're in here, could you change this condition to if-else since there's not a third possibility? It's the cleanest way to complete the conditional analysis.